tests/integration/test_render.py

```diff
@@ -213,6 +214,30 @@ def test_popup(self):
         self.assertEqual(popup_content.text, "")
         self.assertEqual(popup_wrapper.get_attribute('aria-live'), 'polite')
 
+    @data(None, Keys.RETURN)
+    def test_go_to_beginning_button(self, action_key):
+        self.load_scenario()
+        self.scroll_down(250)
+
+        button = self._get_go_to_beginning_button()
+        self.assertEqual(button.get_attribute('tabindex'), '0')
```

Comment:
Why is it important that the tabindex is 0? It seems that you can always tab to the Review button.